configuration/pih/pih-config-haiti-verrettes.json

```diff
@@ -0,0 +1,39 @@
+{
+  "welcomeMessage": "Bienvenue dans l'EMR de Verrettes",
+  "site": "VERRETTES",
+
+  "components": [
+    "chwApp",
+    "clinicianDashboard",
+    "dataExports",
+    "dbevent",
+    "exportPatients",
+    "importPatients",
+    "markPatientDead",
+    "mentalHealth",
+    "myAccount",
+    "patientRegistration",
+    "programs",
+    "systemAdministration",
+    "visitManagement",
+    "visitNote"
+  ],
+
+  "globalProperties": {
+    "coreapps.defaultPatientIdentifierLocation": "6c1c2579-7d16-4d39-bca2-e77141ff435f"
+  },
+
+  "extraIdentifierTypes": [
+    "d9009bd0-eac9-11e5-a837-0800200c9a66"
+  ],
+
+  "registrationConfig": {
+    "afterCreatedActions": [
```

Comment:
We set this blank to override setting in pih-config-haiti.json that tries to generate a dossier number after the patient has been created.